doc/source/methods/ProtoSelect.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<ol><li>"log" -&gt; "logarithm" (I find it easier to read if it's spelled out in written form)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>